ctpolicy/ctpolicy_test.go

```diff
@@ -65,40 +71,52 @@ func TestGetSCTs(t *testing.T) {
 		{
 			name: "basic failure case",
 			mock: &alwaysFail{},
-			groups: [][]cmd.LogDescription{
+			groups: []cmd.CTGroup{
 				{
-					{URI: "abc", Key: "def"},
-					{URI: "ghi", Key: "jkl"},
+					Name: "a",
+					Logs: []cmd.LogDescription{
+						{URI: "abc", Key: "def"},
+						{URI: "ghi", Key: "jkl"},
+					},
 				},
 				{
-					{URI: "abc", Key: "def"},
-					{URI: "ghi", Key: "jkl"},
+					Name: "b",
+					Logs: []cmd.LogDescription{
+						{URI: "abc", Key: "def"},
+						{URI: "ghi", Key: "jkl"},
+					},
 				},
 			},
 			ctx:       context.Background(),
-			errRegexp: regexp.MustCompile("CT log group .: all submissions failed"),
+			errRegexp: regexp.MustCompile("CT log group \".\": all submissions failed"),
```

Comment:
I think this regex is a bit brittle and will break with log names greater than 1 char. Maybe use `\".+\"` instead?

Reply:
Given this is for a fixed test case where the log names are one character I'm not sure what the point would be? (Except for making copy/pasting easier for another test down the road but enabling that doesn't seem like a great idea)

Reply:
I was thinking of someone adding another case to this test. You need to know the unstated assumption that all tests will have single letter names.

It's too small to argue about. I'm happy for it to stay as-is for this test.